lib/account.rb

```diff
@@ -0,0 +1,26 @@
+class Account
```

Comment:
What are the `Account`'s responsibilities? Is there more than one? (For example, have a look at the `transactions` array and its manipulation.